resources/en-US/training/t-z.training.bxb

```diff
@@ -1,10 +1,20 @@
+train (t-z285r9g9qcqbpz6zl3e4xkhhi) {
+  utterance ("[g:CommitOrder:continue,r:UpdateOrder] Remove the {[g:RemovedItem] (polo)[v:SearchTerm]} and the {[g:RemovedItem2] (v-neck)[v:SearchTerm]}")
+  plan (rVHLCsJADBxtra1vK4r1ov6AB3/BkyfBxweUNmChi2W7iie/3fhA7NIqiBs2hJnJJBD00KezL5KYZuk+kmq2poCiRMHLwouDEJFayZAkelnqAWr6XRL6ih7UtJC6HO91ipG+hDicKFwqEikm2iaSuCWj8Aq7Mf7S/GHy/J++G4opUE9umMflM74M9luSv/rlnqp4yAAllGHARAUWqrDhoIY6GmiihTY66L7CBTS19aa2WW1wrnPd4Sjd1Xjm2zc17P1ZjBjs6LJji2dU4VwB)
+  last-modified (1560817227400)
+}
 train (t-zoi8m5pwtqcsfqh32rqro0e3u) {
   utterance ("[g:CommitOrder,r:CreateOrder,r:CreateItem] Buy (4)[v:Quantity:4] (medium)[v:Size:Medium] (sweatshirts)[v:SearchTerm]")
-  plan (dY7LCsIwEEWvj7a+VoqvP3Bjf0IQXInWHwjtQAOmDekI6sJvN21BTNDFzMA5d+BigjndhdJXiqtcGo7PlJLUjLWLd6VSko8mI4OZq1ro5w0JplZNXXVgUlj9ijdm88+8pF1bbUpNhiVVWLjJ000ULPnh90vqjaUL97LIWuE1SUiYNL+QUX7vRD4JY3TQRQ99BAgRYYAhRpbC4UHDQ8vwGXzd2kTNX2B/8AY=)
-  last-modified (1559953579428)
+  plan (dY/LCsIwEEWvr9b3QvH5BW70JwShK9H6A0EHHDC1pCOoC7/d2G5MtItMwjn3wgR9jOiudHqhVXZmI6s9HYlTwczF66vWLFtzIoOhqwro5w0poULNXRUlLKwukZDO/tcKNfBqFmJaFseizLzYjmVqrikZYcowdpO7m0qE5eF/K/5MTFy44eRUCG+TmJQ5ng9ktL93zE9CDxVUUUMdDQQI0UQLbXTQtQY/Lsxd03I4B193YF+tvB/aLt4=)
+  last-modified (1560832167823)
 }
 train (t-zpxxdnt3h425qsxo95s73qmss) {
   utterance ("[g:Shirt:prompt] (Oxford dress shirt)[v:SearchTerm]")
   plan (Y2BmEE6tSMwtyEnVK87ILCrRCwaRDOKogm6ZeSkQCQk01amJRckZIalFuQxMDIxAzAAkGRlAgAmCAQ==)
   last-modified (1559955718525)
+}
+train (t-zyz82pb8k825dgoga5hvrn9pr) {
+  utterance ("[g:CommitOrder:continue,r:UpdateOrder] Change the {[g:RemovedItem] (logo tee)[v:SearchTerm]} to a small {[g:AddedItem] (v-neck)[v:SearchTerm]}")
```

Comment:
tag small here?

Reply:
Actually should not tag small here. Currently can only search in the list using search term (the SelectItem does not take size as input), I tried it with that but then for a change item use case the planner can easily mistake the Size as a searchTerm with the Size as a the new item size.